.travis.yml

```diff
@@ -1,10 +1,19 @@
 language: python
 python:
 - '2.7'
+before_install:
+- wget http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O miniconda.sh
+- chmod +x miniconda.sh
+- ./miniconda.sh -b
+- export PATH=/home/travis/miniconda/bin:$PATH
```

Comment:
to make this work on our computers, install it next to appegine, and not rely on default path (/travis/...), you could give the installation path, something like ./miniconda.sh -b -p ./lib